src/ui/rcb-ui-inner.svelte

```diff
@@ -36,6 +36,14 @@
   export let purchaseOperationHelper: PurchaseOperationHelper;
   export let isInElement: boolean = false;
 
+  const shouldShowDetailsButton =
+    (Boolean(brandingInfo?.appearance) &&
+      Boolean(productDetails?.description)) ||
+    Boolean(
+      (purchaseOptionToUse as SubscriptionOption)?.trial?.periodDuration,
+    ) ||
+    false;
```

Comment:
I think we can avoid this false?

```suggestion
    );
```

Reply:
sure I have have casting to boolean on the otherones I don't think I do, will remove